### DISCUSSION TITLE
Is it possible to play the video using these files in index.html?

### DISCUSSION THREAD

**Discussion by MetakingGGH** [community/community#19523] [Actions] [answered]
hello. I am a beginner in programming and currently studying deep reinforcement learning using MAgent.
(https://github.com/geek-ai/MAgent/)

After doing the calculations, I got the two files named "config.json" and "video_1.txt".

*Since you cannot upload a file in 'json' format here, save the contents of the config.json file as txt and upload it.

[config.txt](https://github.com/github-community/community/files/8988108/config.txt)

[video_1.txt](https://github.com/github-community/community/files/8988107/video_1.txt)

[video_2.txt](https://github.com/github-community/community/files/8988118/video_2.txt)

'MAagent' says that it can play video using these, but I don't know how to proceed.

It seems to serve an index.html file, but I can't find it.
![image](https://user-images.githubusercontent.com/108246531/175846443-638dd541-96c5-4433-ad47-f16ad2121fb7.png)

If anyone knows how to play these, please let me know.

**Comment by melvincwng** [comment, marked as answer]
@MetakingGGH, perhaps you can try creating an issue and contact the creators of that particular library.

Hope this helps.

**Reply by MetakingGGH** [reply]
Thanks for the kind advice. I will follow your advice.

**Reply by melvincwng** [reply]
@MetakingGGH, no problem! Pleasure in helping you.

If this has been helpful, please kindly 'Mark as answer' so that the rest are aware this issue has been addressed.

Thank you for your cooperation, and have a pleasant day ahead 😊

@MetakingGGH, btw, if you would like to stay in touch, feel free to follow me on GitHub :).
